release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -1944,10 +1944,10 @@ In the table below, features are marked with the following statuses:
 |TP
 |GA
 
-|Memory Manager feature
+|Memory Manager
 |-
 |-
-|TP
+|GA
```

Comment:
I don't know the proper way so let me ask: shouldn't it be TP in the previous version?

Reply:
@fromanirh Based on the bug's description, my understanding was that this feature was supposed to be in GA status in 4.9 as well. Link to original bug: https://bugzilla.redhat.com/show_bug.cgi?id=2105152
@fbaudin, you've lgtm'd this change, could you please clarify?